actors/builtin/paych/paych_actor.go

```diff
@@ -154,10 +156,14 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 		rt.Abortf(exitcode.ErrIllegalArgument, "voucher signature invalid")
 	}
 
-	if rt.CurrEpoch() < sv.TimeLock {
+	if rt.CurrEpoch() < sv.TimeLockMin {
 		rt.Abortf(exitcode.ErrIllegalArgument, "cannot use this voucher yet!")
 	}
 
+	if rt.CurrEpoch() > sv.TimeLockMax {
+		rt.Abortf(exitcode.ErrIllegalArgument, "this voucher has expired!")
+	}
```

Comment:
Please add a test exercising this check. Probably another case next to the one labelled "fails if too early for voucher"

Reply:
Done, thank you!